ci-tools/fpga-boss/src/ftdi.rs

```diff
@@ -24,10 +24,12 @@ pub enum BitMode {
     BitBang = 0x01,
     CBus = 0x20,
 }
+
 pub struct FtdiCtx {
     ctx: *mut libftdi1_sys::ftdi_context,
     port_path: UsbPortPath,
 }
+
```

Comment:
nit: Prefer to omit whitespace changes

Reply:
No this was no cargo fmt change, I will revert it.